recipe/meta.yaml

```diff
@@ -13,15 +13,15 @@ source:
     - tzdata-location.patch
 
 build:
-  number: 5
-  skip: true  # [py>39]
+  number: 0
+  skip: true  # [py>38]
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - patch # [not win]
+    - m2-patch # [win]
   host:
     - backports
     - python
```

Comment:
Could you add please `setuptools >=40.8.0` and `wheel`? See https://github.com/pganssle/zoneinfo/blob/4299bfd7623934acc6f27b32744b02a147912120/pyproject.toml#L2

Reply:
Thank you - I added these modules to `build`

Reply:
`>=40.8.0` without a whitespace should be